src/app/admin/coupon/register/page.tsx

```
@@ -223,61 +92,107 @@ export default function CouponRegisterPage() {
-        setCouponEndDateTime(new Date(couponEndDateTime.setHours(hour)))
-    }
-
-    const handleCouponExpireDateTimeChange = (hour: number) => {
-        setCouponExpireDateTime(new Date(couponExpireDateTime.setHours(hour)))
-    }
-
+    // 기간 조정
     useEffect(() => {
-        handleCouponPeriodChange(couponStartDateTime, couponEndDateTime)
-    }, [couponStartDateTime, couponEndDateTime])
+        setCouponEndDateTime(getValidatedDateHour(couponEndDateTime, couponStartDateTime, COUPON_EVENT_START_TO_END_MIN_HOUR))
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: couponEndDateTime

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)